modules/learn/pages/data/durability.adoc

```diff
@@ -216,16 +216,34 @@ If a durable write is attempted on a bucket that has been configured with three
 
 xref:learn:clusters-and-availability/automatic-failover.adoc[Automatic Failover] removes a non-responsive node from the cluster automatically, following an administrator-configured timeout.
 Active vBuckets thereby lost are replaced by the promotion of replica vBuckets, on the surviving nodes.
-A maximum of three sequential automatic failovers can be configured to occur.
+The maximum number of sequential automatic failovers is configurable by the administrator.
 
 In cases where commitment based on _persistToMajority_ has occurred, but no further propagation of the new data across the cluster has yet occurred, automatic failover of the nodes containing the new data results in the data's loss &#8212; since no updated replica vBucket yet exists elsewhere on the cluster.
 
 For example, if a bucket has two replicas, the total number of nodes on which the data resides is _three_; and the _majority_ of nodes, on which persistence must occur prior to commitment, is _two_.
 After commitment, if those two nodes become unresponsive, automatic failover, if configured to occur up to a maximum of two times, allows those two nodes to be failed over _before_ the durable write has been made persistent on the third node.
 In such a case, the durable write is lost, and the success message already delivered to the application rendered false.
 
-To prevent this, and thereby maintain guaranteed protection, at least one of the unresponsive nodes containing the new data should _not_ be failed over.
-Therefore, auto-failover should be configured to occur sequentially only up to the number of times that supports this requirement.
+To prevent this, and thereby maintain guaranteed protection, at least one of the unresponsive nodes containing the new data should _not_ be failed over; and auto-failover should be allowed to occur sequentially only up to the number of times that supports this requirement.
+This may be best achieved as described immediately below, in xref:learn:data/durability.adoc#preserving-durable-writes[Preserving Durable Writes].
+
+[#preserving-durable-writes]
+==== Preserving Durable Writes
+
+In Couchbase Enterprise Server 7.2+, auto-failover can be configured not to occur if a node's failover would result in the loss of durably written data.
+This means ensuring that at least one node containing the durably written data is preserved.
+
+If auto-failover is configured in this way:
+
+* If data has been durably written to _one_ node (which is to say, no replica exists), the node cannot be automatically failed over.
```

Comment:
In the case in which we have 0 replicas a failover is possible. A different warning is presented, related to failing over a node without replicas:

<img width="528" alt="Screenshot 2023-07-04 at 11 05 19" src="https://github.com/couchbase/docs-server/assets/40238853/677b728e-8eb1-46c9-ac8d-9941791b69ed">